src/whylogs/core/statistics/constraints.py

```diff
@@ -31,6 +32,8 @@
     Op.NE: lambda x: lambda v: v != x,
     Op.GE: lambda x: lambda v: v >= x,
     Op.GT: lambda x: lambda v: v > x,  # assert incoming value 'v' is greater than some fixed value 'x'
+    Op.MATCH: lambda x: lambda v: re.match(x, v) is not None,
```

Comment:
Does re.compile also work here?

Reply:
yes, missed this. Would be good to compile the regex once when constructing the in-memory constraint.